main/build.gradle.kts

```
@@ -0,0 +1,44 @@
+/*
+ * Designed and developed by Wedemy 2023.
+ *
+ * Licensed under the MIT.
+ * Please see full license: https://github.com/Wedemy/eggeum-android/blob/main/LICENSE
+ */
+@file:Suppress("UnstableApiUsage", "INLINE_FROM_HIGHER_PLATFORM")
+
+
+plugins {
+  eggeum("android-library")
+  eggeum("android-hilt")
+  alias(libs.plugins.google.secrets)
+  alias(libs.plugins.androidx.navigation.safeargs)
+}
+
+android {
+  namespace = "us.wedemy.eggeum.main"
```

```suggestion
  namespace = "us.wedemy.eggeum.android.main"
```

`android` 까지 들어가는 게 우리 앱 패키지명이에요!

---

> `android` 까지 들어가는 게 우리 앱 패키지명이에요!

그렇군요 궁금한 부분이었는데 다 추가해놔야겟어요